package.json

```diff
@@ -64,6 +65,7 @@
     "babel-plugin-tester": "^5.0.0",
     "babel-preset-env": "^1.6.1",
     "babel-preset-flow": "^6.23.0",
+    "babel-preset-react": "^6.24.1",
```

Comment:
remove

Reply:
I found that without this, jest was unable to run my component fixtures. It throws errors when it finds the JSX nodes